angular-workspace/projects/ni/nimble-angular/src/directives/combobox/tests/nimble-combobox-control-value-accessor.directive.spec.ts

```diff
@@ -225,6 +225,32 @@ describe('Nimble combobox control value accessor', () => {
 
             expect(testHostComponent.callbackValue).toEqual(OPTION_NOT_FOUND);
         });
+
+        it('update option value with new model, followed by updating combobox value with same model updates display correctly', async () => {
+            const newModelValue: TestModel = { name: 'newName', value: 100 };
+            testHostComponent.dynamicOption = newModelValue;
+            testHostComponent.selectedOption = newModelValue;
+            fixture.detectChanges();
+            await waitForUpdatesAsync();
+
+            expect(combobox.value).toBe('newName');
+        });
+
+        it('combobox supports a large amount of options', () => {
+            const newOptions: TestModel[] = [];
+            for (let i = 0; i < 300; i++) {
+                newOptions.push({ name: i.toString(), value: i });
+            }
+            testHostComponent.selectOptions = newOptions;
+            let exceptionThrown = false;
+            try {
+                fixture.detectChanges();
+            } catch (e) {
+                exceptionThrown = true;
+            }
+
+            expect(exceptionThrown).toBeFalse();
```

Comment:
prefer the jasmine exception helpers

```suggestion
    expect(() => {
        fixture.detectChanges();
    }).not.toThrow();
```